util/autogen_testutils/isr_testutils.h.tpl

```diff
@@ -0,0 +1,29 @@
+${license_banner}
+
+${autogen_banner}
+
+#ifndef OPENTITAN_SW_DEVICE_LIB_TESTING_AUTOGEN_ISR_TESTUTILS_H_
+#define OPENTITAN_SW_DEVICE_LIB_TESTING_AUTOGEN_ISR_TESTUTILS_H_
+
+% for ip in ips_with_difs:
+  % if ip.irqs:
+  /* 
+   * Services an ${ip.name_snake} IRQ at the IP that raised it, and verifies this
+   * matches the IRQ that was raised at the PLIC.
+   *  
+   * @param ${ip.name_snake} A(n) ${ip.name_snake} DIF handle. 
+   * @param plic_irq_id The triggered PLIC IRQ ID. 
+   * @param plic_${ip.name_snake}_start_irq_id The PLIC IRQ ID where ${ip.name_snake} starts. 
+   * @param expected_${ip.name_snake}_irq The expected ${ip.name_snake} IRQ. 
+   * @param is_only_irq This is the only IRQ expected to be raised.
+   */
+  void isr_testutils_${ip.name_snake}(dif_${ip.name_snake}_t *${ip.name_snake}, 
```

Comment:
What about ${ip.name_snake}_isr_testutils?

Reply:
The way it is keeps the convention of: <IP or functionality lib name>_testutils_<function name>, described in #7976, where "<IP or functionality lib name>_testutils" is supposed to match the testutils file name.